spec/registry.md

```diff
@@ -1109,6 +1109,31 @@ the functions `:datetime` and `:time`:
   - `true`
   - `false`
 
+The following _option_ and its values will be REQUIRED to be available on
+the functions `:datetime`, `:date`, and `:time`.
+This option currently has a Maturity Level of **Proposed**.
+
+- `timezone`
+  - A valid time zone identifier
+    (see [BCP175](https://www.rfc-editor.org/bpc/bpc175))
+  - `local`
+  - `UTC`
+  
+A time zone identifier is well-formed if it matches `tzId` in the following ABNF:
```

Comment:
As discussed during the call, it is not appropriate for us to include this ABNF, as we cannot require that IANA or any other external authority will fit their valid values within it.

Reply:
Agreed.